SentinelOne/manifest.json

```
@@ -22,5 +22,5 @@
     "name": "SentinelOne",
     "uuid": "ff675e74-e5c1-47c8-a571-d207fc297464",
     "slug": "sentinelone",
-    "version": "1.14.1"
+    "version": "1.15.0"
```

You should increase the version number for each PR.
```suggestion
    "version": "1.15.1"
```